sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -386,6 +389,8 @@ CosmosKeyCredential cosmosKeyCredential() {
      */
     public CosmosPagedFlux<CosmosDatabaseProperties> queryDatabases(SqlQuerySpec querySpec, FeedOptions options) {
         return UtilBridgeInternal.createCosmosPagedFlux(pagedFluxOptions -> {
+            String spanName = "queryDatabases." + querySpec.getQueryText();
```

Comment:
this logs the query text? If the query has private info, this will get logged? is that ok?